play.pokemonshowdown.com/src/battle-log.ts

```diff
@@ -1788,3 +1789,11 @@ export class BattleLog {
 		return 'data:text/plain;base64,' + encodeURIComponent(btoa(unescape(encodeURIComponent(replayFile))));
 	}
 }
+
+const net = window.Net as typeof Net | undefined;
+if (net) {
+	net(`/config/colors.json`).get().then(response => {
```

Comment:
```suggestion
if (window.Net) {
	Net(`/config/colors.json`).get().then(response => {
```

I'd just do this tbh.

Reply:
Does optional chaining not compile well here?

Reply:
I think we have `window: any` declared, so it at least wouldn't be typesafe the way this one is.

Reply:
And "not typesafe" means "TS will force us to declare the type of `response`"

Reply:
Ah, so because the original code had declared the type of `net` it could have used ``net?.(`/config/colors.json`).get().then(response => {`` (but not having to declare the type in the first place is better still of course).